spec/examples/error/lib/error.ex

```diff
@@ -0,0 +1,4 @@
+defmodule Error do
+	def go do
+	# missing end
+end
```

Comment:
The Elixir code that triggers a compiler error. One folder per compiler msg we are testing.